api/src/main/java/org/pmiops/workbench/db/model/DbInstitution.java

```diff
@@ -95,66 +85,29 @@ public DbInstitution setOrganizationTypeOtherText(String organizationTypeOtherTe
     return this;
   }
 
-  @OneToMany(mappedBy = "institution", cascade = CascadeType.ALL)
-  @NotNull
-  public Set<DbInstitutionEmailDomain> getEmailDomains() {
-    return emailDomains;
-  }
-
-  /**
-   * Effectively: do an in-place this.emailDomains = emailDomains
-   *
-   * <p>Hibernate doesn't like it when you reassign collections. Instead, modify in-place. First,
-   * call retainAll() to subset DB rows to those we wish to keep: the intersection of old and new.
-   * Then, call addAll() to add the diff(new - old) rows.
-   *
-   * <p>https://stackoverflow.com/questions/5587482/hibernate-a-collection-with-cascade-all-delete-orphan-was-no-longer-referenc
-   *
-   * @param emailDomains the new collection of domains for this Institution
-   */
-  public DbInstitution setEmailDomains(final Collection<DbInstitutionEmailDomain> emailDomains) {
-    final Set<DbInstitutionEmailDomain> attachedDomains =
-        Optional.ofNullable(emailDomains)
-            .map(Collection::stream)
-            .orElse(Stream.empty())
-            .map(domain -> domain.setInstitution(this))
-            .collect(Collectors.toSet());
-    // modifies this set so that its value is the intersection of the two sets
-    this.emailDomains.retainAll(attachedDomains);
-    this.emailDomains.addAll(Sets.difference(attachedDomains, this.emailDomains));
+  // omit ID field from equality so equivalent objects match regardless
```

Comment:
An earlier version of this was at #3601

Reply:
Are all of our Db objects using the same semantics for `equals()`?

Reply:
I did not audit other DB objects.  In the case of DbInstitution and friends, it is necessary for things like determining Set membership and comparisons between already-persisted and newly-created objects.